### PR TITLE
Fix fetcher settings html import

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
Fixes a bad import of the html template for fetcher settings

**Issues**
Fixes https://github.com/jellyfin/jellyfin-web/issues/2114

### CODE REVIEW

**Review by MrTimscampi — APPROVED** [review]

**Review by dmitrylyzo — APPROVED** [review]

File: src/components/imageOptionsEditor/imageOptionsEditor.js [hunk 1]
```diff
@@ -90,9 +91,6 @@ import '../../elements/emby-input/emby-input';
     }
 
     async function showEditor(itemType, options, availableOptions) {
```
Comment (Camc314) [inline src/components/imageOptionsEditor/imageOptionsEditor.js:92]:
```suggestion
    function showEditor(itemType, options, availableOptions) {
```

Comment (Camc314) [reply src/components/imageOptionsEditor/imageOptionsEditor.js:92]: Double check (I'm on mobile) But I can't see any awards any more

Comment (dmitrylyzo) [reply src/components/imageOptionsEditor/imageOptionsEditor.js:92]: it should probably return `Promise` because it is exported as `show`.

Comment (thornbill) [reply src/components/imageOptionsEditor/imageOptionsEditor.js:92]: Removed!

**Pull request merged**